packages/iproute2.rb

```diff
@@ -0,0 +1,61 @@
+# Adapted from Arch Linux iproute2 PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/iproute2/trunk/PKGBUILD
+
+require 'package'
+
+class Iproute2 < Package
+  description 'IP Routing Utilities'
+  homepage 'https://git.kernel.org/pub/scm/network/iproute2/iproute2.git'
+  version '6.1.0'
+  license 'GPL2'
+  compatibility 'all'
+  source_url 'https://www.kernel.org/pub/linux/utils/net/iproute2/iproute2-6.1.0.tar.xz'
+  source_sha256 '5ce12a0fec6b212725ef218735941b2dab76244db7e72646a76021b0537b43ab'
+
+  binary_url({
+    i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/iproute2/5.16.0_i686/iproute2-5.16.0-chromeos-i686.tar.zst',
```

Comment:
Binary doesn't match version?

Reply:
Removed binary... since dependency iptables doesn't build for i686...